app/(candidate)/dashboard/website/create/components/WebsiteCreateFlow.js

```diff
@@ -32,7 +32,7 @@ export default function WebsiteCreateFlow() {
     const saved = await handleSave()
 
     if (saved) {
-      router.push('/dashboard/website')
+      window.location.href = '/dashboard/website'
```

Comment:
Seems odd to be doing a window page-level address change and page-load here, but still using the `router` to make navigation changes elsewhere in this component? 🤔 

https://github.com/thegoodparty/gp-webapp/pull/799/files?w=1#diff-568754ec22c7ef63adf11a656decf1342ecdb43dff894747a55b7e2901c9646aR105

Why this change?